src/scripts/market.rs

```diff
@@ -26,15 +26,14 @@ pub async fn download_and_display_offers(
     let offers_addresses =
         sync_offerbook_with_addresses(maker_addresses.clone(), &TakerConfig::default()).await;
 
-    // TODO: This is too verbose to just print stuffs. Shorten it in a single `iter().for_each()`.
```

Comment:
The TODO was meant for the whole following section. Not just this iter. I don't fully understand why it was written this way. And we will repurpose the script module later anyway. 

So ACK for me on this change. 

But it's just a print, so the whole section after the comment, could have been a simple one-liner. 

But not worth spending time on it now.